docs/topic/billing/index.md

```diff
@@ -0,0 +1,11 @@
+# Cloud Billing
+
+These topics provide us foundational common shared knowledge and
+terminology to talk about cloud billing.
+
+```{toctree}
+:maxdepth: 2
+things
```

Comment:
```suggestion
chargeable-products
```

I recommend renaming this file to something like chargeable-products so it's a bit more descriptive than `things` (I'm willing to be told 'product' isn't right here, even naming the file `what-we-are-charged-for` would be an improvement, I think)

Reply:
I've renamed it to `chargeable-resources`